bool capture
string capture_Place
string next_Move
---
